fedmsg_meta_fedora_infrastructure/autocloud.py

```diff
@@ -106,9 +110,10 @@ def handle_v2_autocloud_link(self, msg, **config):
             else:
                 return msg['msg']['image_url']
         elif 'autocloud.compose' in msg['topic']:
-            compose_job_id = msg['msg']['compose_job_id']
-            return 'https://apps.fedoraproject.org/autocloud/jobs/%s' % (
-                compose_job_id)
+            compose_job_id = msg['msg'].get('compose_job_id')
+            if compose_job_id is not None:
+                return 'https://apps.fedoraproject.org/autocloud/jobs/%s' % (
+                    compose_job_id)
```

Comment:
Since I am not returning anything it should return None. If there is no compose_job_id then it should not create any URL.